fcc-test-suite-template-master/css/style.css

```diff
@@ -11,4 +11,336 @@
     }
```

Comment:
This applies to the entire stylesheet document.
Take note that you need a maximum indentation of 2 spaces or stylelint will flag errors.
See http://stylelint.io/user-guide/rules/indentation/ for more information.